lib/__tests__/cloudworker-e2e.test.js

```diff
@@ -211,4 +211,186 @@ describe('cloudworker-e2e', async () => {
     await upstream.close()
     cb()
   })
+
+  test('simple sha256 encode', async (cb) => {
+    const script = `
+      addEventListener('fetch', event => {
```

Comment:
For long e2e scripts like this, please add as a fixture file instead. It helps with readability.